apps/usage_tracking/tests/test_entity_extractor.py

```
@@ -55,3 +55,25 @@ def test_extract_mixed_items_skips_non_dict_entries(self):
+        from apps.usage_tracking.services.entity_extractor import extract_entities
+
+        body = json.dumps([{"id": 1, "name_en": "Ibn Kathir"}, {"id": 2, "name": "Al-Sudais"}]).encode()
+        ids, names = extract_entities(body)
+        assert ids == [1, 2]
+        assert names == ["Ibn Kathir", "Al-Sudais"]
+
+    def test_extract_name_falsy_string_zero_is_returned(self):
+        from apps.usage_tracking.services.entity_extractor import extract_entities
+
+        body = json.dumps([{"id": 1, "name": "0"}]).encode()
+        ids, names = extract_entities(body)
```

Ruff (0.15.12)

[warning] Unpacked variable `ids` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)